recipes/opengl/all/conanfile.py

```diff
@@ -36,12 +39,15 @@ def system_requirements(self):
         pacman.install(["libglvnd"], update=True, check=True)
 
         zypper = package_manager.Zypper(self)
-        zypper.install_substitutes(["Mesa-libGL-devel", "glproto-devel"], 
+        zypper.install_substitutes(["Mesa-libGL-devel", "glproto-devel"],
                                    ["Mesa-libGL-devel", "xorgproto-devel"], update=True, check=True)
 
         pkg = package_manager.Pkg(self)
         pkg.install(["libglvnd"], update=True, check=True)
 
+        pkg_util = package_manager.PkgUtil(self)
```

Comment:
I may be missing something in the PR description but I can't find any details provided for this change?

Reply:
It's the last sentence of PR description, about SunOs

Reply:
thanks! I knew i was missing something